hw_3.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+//NORMAL level
+
+//Task 1
+let tempVar1 = 'true';
+let tempVar2 =  false;
+let tempVar3 = 17;
+let tempVar4 = undefined;
+let tempVar5 = null;
+let tempVar;
+
+// for (let i = 1; i <= 5; i++) {
+//     tempVar = tempVar + i;
+//     console.log(`Variable: tempVar${i} have type: ${typeof tempVar}`); 
+// } 
+
+console.log(`Variable: tempVar1 have type: ${typeof(tempVar1)}`);
+console.log(`Variable: tempVar2 have type: ${typeof(tempVar2)}`);
+console.log(`Variable: tempVar3 have type: ${typeof(tempVar3)}`);
+console.log(`Variable: tempVar4 have type: ${typeof(tempVar4)}`);
+console.log(`Variable: tempVar5 have type: ${typeof(tempVar5)}`);
+
+//Task 2
+let height = 15;
+let width = 20;
+
+console.log(height > width ? height : width);
+
+//Task 3
+for (let i = 1; i <= 20; i++){
+    if(i%3 !== 0){
```

Comment:
и в остальных местах поправь пожалуйста
```suggestion
    if(i % 3 !== 0){
```